src/corelibs/U2View/src/ov_msa/phy_tree/MSAEditorMultiTreeViewer.cpp

```diff
@@ -50,7 +50,8 @@ MSAEditorMultiTreeViewer::MSAEditorMultiTreeViewer(const QString& title, MSAEdit
 
     this->setLayout(treeAreaLayout);
 
-    connect(treeTabArea, SIGNAL(si_tabsCountChanged(int)), SIGNAL(si_tabsCountChanged(int)));
+    connect(treeTabArea, &MsaEditorTreeTabArea::si_tabsCountChanged, this, &MSAEditorMultiTreeViewer::si_tabsCountChanged);
+    connect(treeTabArea, &MsaEditorTreeTabArea::si_activeTabChanged, this, [this] { emit si_activeTreeViewChanged(); });
```

Comment:
Are you sure, that you need an extra lambda here? As far as I remember, it's possible just to set a signal as the second argument and it will be proxied. But, I used it when we used `SIGNAL` or `SLOT` macros, I'm not 100% sure, that it work in this new format

Reply:
Yes I know it works with SIGNAL/SLOT but not sure if it works with raw function pointers.